src/bookmark.cpp

```diff
@@ -194,7 +205,8 @@ Bookmark Bookmark::Deserialize(const QByteArray& data)
             >> result.m_Tags
             >> result.m_AddTime
             >> result.m_UpdateTime
-            >> status;
+            >> status
+            >> result.m_Content;
```

Comment:
use new version for reading new value from cache
if (ver == 2) 
{
in >> result.m_Content;
}